pkg/controllers/deprovisioning/types.go

```diff
@@ -136,38 +136,30 @@ func (c *Candidate) lifetimeRemaining(clock clock.Clock) float64 {
 	return remaining
 }
 
-type action byte
+type Command struct {
+	candidates   []*Candidate
+	replacements []*scheduling.Machine
+}
 
 const (
-	actionDelete action = iota
-	actionReplace
-	actionDoNothing
+	noOpAction = "no-op"
```

Comment:
If we're going to have defined values like this, we should probably make a type

Reply:
I'm not sure I agree here. These are the only three constants that we’re using. A type here feels like unnecessary abstraction. WDYT